Source/player.h

```diff
@@ -635,7 +635,7 @@ struct Player {
 	void RestorePartialLife();
 
 	/**
-	 * @brief Resets hp to maxHp
+	 * @brief Resets hp to maxHitPoints

```

Comment:
I don't think you changed this?

Reply:
Oh, I must have gone a little too far with automatic renaming. Fixed